src/renate/utils/misc.py

```diff
@@ -37,3 +39,45 @@ def maybe_populate_mask_and_ignore_logits(
         logits.index_fill_(1, class_mask.to(logits.device), -float("inf"))
 
     return logits, class_mask
+
+
+class AdditionalTrainingMetrics(Callback):
+    def __init__(self) -> None:
+        self._training_times = []
+
+    def on_train_start(self) -> None:
+        self._training_times.append(time.time())
+
+    def on_train_epoch_end(self) -> None:
+        self._training_times.append(time.time())
+
+    def __call__(self, model: torch.nn.Module) -> Dict[str, Union[float, int]]:
+        if self._training_times:
+            epoch_training_time = self._training_times[-1] - self._training_times[0]
```

Comment:
why is this the epoch_training_time? isn't this total training time?

Reply:
Modifying.